File: devcamp2018/app/src/main/java/com/psato/devcamp/presentation/home/HomeFragmentViewModel.java
package com.psato.devcamp.presentation.home;

import android.arch.lifecycle.ViewModel;
import android.content.Intent;
import android.view.View;

import javax.inject.Inject;

public class HomeFragmentViewModel extends ViewModel {

    @Inject
    public HomeFragmentViewModel() {
    }

    public void onMVVMClicked(View view) {
        view.getContext().startActivity(new Intent(view.getContext(), com.psato.devcamp.presentation.MVVM.QueryActivity.class));
    }
}

File: devcamp2018/app/src/main/java/com/psato/devcamp/presentation/home/HomeFragmentViewModel.kt
package com.psato.devcamp.presentation.home

import android.arch.lifecycle.ViewModel
import android.content.Intent
import android.view.View

import javax.inject.Inject

class HomeFragmentViewModel @Inject
constructor() : ViewModel() {

    fun onMVVMClicked(view: View) {
        view.context.startActivity(Intent(view.context, com.psato.devcamp.presentation.MVVM.QueryActivity::class.java))
    }
}
